src/app/address/controllers/address/manual.js

```diff
@@ -29,26 +32,36 @@ class AddressController extends BaseController {
       if (req.url.includes("edit")) {
         values.checkDetailsHeader = "false";
       }
+
+      addIndividualFieldErrorMessages(
+        "ukBuildingAddressEmptyValidator",
+        req,
+        values
+      );
+
+      values.ukBuildingAddressEmptyValidator = this.isBuildingAddressEmpty(
```

Comment:
Please could we call this something like ukBuildingAddressEmptyError? It's not actually a validator is it?